charts/ccd-case-management-web/values.preview.template.yaml

```diff
@@ -6,19 +6,20 @@ nodejs:
   ingressHost: ${SERVICE_FQDN}
   environment:
     IDAM_LOGIN_URL: https://idam-web-public.aat.platform.hmcts.net
-    CCD_GW_LOGOUT_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/logout
-    CCD_API_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/aggregated
-    CCD_DATA_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/data
-    CCD_GW_OAUTH2_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/oauth2
-    DM_URL: https://ccd-api-gateway-web-pr-273.service.core-compute-preview.internal/documents
+    CCD_GW_LOGOUT_URL: https://gateway-ccd.aat.platform.hmcts.net/logout
```

Comment:
Hi Dan, Are the changes to the url configs required? The **pr-273** configs ensure that the preview environments for case-management-web PRs use the back-end of the **develop** branch. Without these, we would not be able to import definition files into the preview environments and get the functional tests passing for the PR.

Reply:
Addressed 👍